README.md

```diff
@@ -6,7 +6,7 @@ The main focus of the package is to transform the core ad object tables into ana
 
 ## Models
 
-This package contains transformation models that are designed to work simultaneously with our [Microsoft Advertising source package](https://github.com/fivetran/dbt_microsoft_ads_source). A dependency on the source package is declared in this package's `packages.yml` file, so it will automatically download when you run `dbt deps`. The primary outputs of this package are described below.
+This package contains transformation models that are designed to work simultaneously with our [Microsoft Advertising source package](https://github.com/fivetran/dbt_microsoft_ads_source) and our [multi-platform Ad Reporting package](https://github.com/fivetran/dbt_ad_reporting). A dependency on the source package is declared in this package's `packages.yml` file, so it will automatically download when you run `dbt deps`. The primary outputs of this package are described below.
```

Comment:
i also added a bit about the ad reporting package on [line 5](https://github.com/fivetran/dbt_linkedin/pull/9/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R5), since it's alluded to there. thoughts on mentioning it in two places in the readme?

Reply:
I really like the added reference on line 5. More promotion for the ad roll up package!